### ISSUE TITLE
User Guide Part 1 notebook's example nwb file does not pass validation to be uploaded

### ISSUE THREAD

**Issue by sooyounga** [dandi/example-notebooks#19]
I get a validation error for the nwb file I created as part of following through the User Training video (using DANDI User Guide Part 1) and could not upload it onto the staging dandiset I created, the command is `dandi upload -i dandi-staging` and this is the error message `sub-001/sub-001_ecephys.nwb 225.0 kB       1              skipped          failed validation  `

Has there been a recent change that prevents this file from passing validation now? Or is there something else I should be doing?
Thank you!

**Comment by djarecka** [comment]
just to add to this: validator gives `Subject.sex is missing.`.  Looks like the requirements was introduced here: https://github.com/NeurodataWithoutBorders/nwbinspector/pull/247

**Comment by djarecka** [comment]
@sooyounga - perhaps you can try to fix the notebook by adding `sex` field to the subject description?

**Comment by sooyounga** [comment]
yes adding `sex` field to the subject worked! using the newly updated NWB Inspector v0.4.18 from https://github.com/dandi/dandi-cli/issues/1031 

I made a tiny PR here for this addition https://github.com/dandi/example-notebooks/pull/20
